source/Keystone/ApplicationCredentials.rst

```diff
@@ -7,6 +7,9 @@ without embedding the main user's password in the application configuration.
 
 Application credentials can be created either through the web interface or command line
 
+.. note::
+
+  Here we assume that users are familiar with using ``clouds.yaml``.
```

Comment:
You need to put a link to the existing doc page, like this https://github.com/stfc/cloud-docs/blob/master/source/ClusterAPI/Setup.rst#cloudsyaml-prep 
```
:ref:`clouds_yaml`
```